### DISCUSSION TITLE
Question about the design of backend

### DISCUSSION THREAD

**Discussion by JackKaiXing** [google/filament#5381] [Q&A] [answered]
Hello, 

Why is filament backend not implemented by using `glew` and `glfw`, but instead using `bulegl` to load opengl functions and creating rendering context by specific platforms(like `platformCocoaGL` and `platformWGL` )?

Thanks.

**Comment by romainguy** [comment, marked as answer]
Because those dependencies are way bigger than what we wanted to take on and not useful to us.
